integration_tests/gotenberg/gotenbergIntegration.test.ts

```diff
@@ -12,8 +10,9 @@ describe('Gotenberg API integration', () => {
   describe('Render a PDF file from HTML', () => {
     it('should render a PDF', async () => {
       const pdf = await client.renderPdfFromHtml(htmlString)
-      const pdfText = await pdfParse(pdf)
-      expect(pdfText.text).toContain('A document')
+      const parser = new PDFParse(pdf)
+      const result = await parser.getText()
+      expect(result.text).toContain('A document')
```

Comment:
API for this changed quite a bit.. I'd like to ditch this dependency as it's only used for tests but didn't see anything better